tests/DashJS/DashJS.ts

```diff
@@ -0,0 +1,12 @@
+import { expect } from 'chai';
```

Comment:
I usually prefer to put the test-files near to the code files and put *.test.* in the name. In that way it is easier to maintain them. 
Furthermore the typescript compile cannot build the project, since it expects all src files to be under the `src` folder: 


```
error TS6059: File '/Users/davidhahn/dev/dash/DashJS/tests/DashJS/DashJS.ts' is not under 'rootDir' '/Users/davidhahn/dev/dash/DashJS/src'. 'rootDir' is expected to contain all source files.

error TS6059: File '/Users/davidhahn/dev/dash/DashJS/tests/DashJS/Platform/Platform.ts' is not under 'rootDir' '/Users/davidhahn/dev/dash/DashJS/src'. 'rootDir' is expected to contain all source files.

error TS6059: File '/Users/davidhahn/dev/dash/DashJS/tests/DashJS/SDK/SDK.ts' is not under 'rootDir' ```
/Users/davidhahn/dev/dash/DashJS/src'. 'rootDir' is expected to contain all source files.

error TS6059: File '/Users/davidhahn/dev/dash/DashJS/tests/fixtures/dp1.schema.json' is not under 'rootDir' '/Users/davidhahn/dev/dash/DashJS/src'. 'rootDir' is expected to contain all source files.
```

Reply:
Valid point, makes sense in TS world (tho I saw *.spec.* used more). I wanted to keep the same dir structure we share in Dashevo repo. 
However, you are right, will fix it.

Reply:
btw. if you still have issues with the rootDir try to remove the `rootDir` prop from tsconfig.json. tsc should be smart enough

Reply:
Should be fixed. Nah, that's the thing, my conf actually did not spot this error. 
So I will need you to confirm this is revolved :)

Reply:
Have still issues with files not being under root dir. Will commit a change on tsconfig.json